resources/static/css/app.scss

```diff
@@ -499,7 +499,12 @@ section.content {
             line-height: 21px;
             max-width: 630px;
             margin: auto;
-            font-size: 50%;
+            font-size: 70%;
+            text-align: center;
+
+            i {
+                color: color(text-blue)
```

Comment:
Could you add a `cursor: pointer;` rule?

Reply:
So it looks really clickable

Reply:
Sure will make the change